tools/osx-uninstall.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# Added to help for removal of node on OSX
```

Comment:
Nit, maybe `# Remove Node.js installed from tarballs or installer on macOS.`

Does this work for other install methods like `brew`?

Also, as apple seems to have decided to go with `macOS` rather than OSX, maybe change the name to `macos-uninstall.sh`?